nsis/src/nsis/NsisAnalyzer.java

```diff
@@ -145,4 +148,44 @@ private void resolveStrings(Instruction instr, MemoryBlock stringsBlock)
 		}
 	}
 
+	/**
+	 * Resolve the control flow for the specified instruction
+	 * 
+	 * @param instr        the instruction
+	 * @param entriesBlock the memory block containing the instructions
+	 * @param program
+	 * @throws MemoryAccessException
+	 */
+	private void resolveControlFlow(Instruction instr, MemoryBlock entriesBlock,
+			ReferenceManager referenceManager) throws MemoryAccessException {
+		String mnemonic = instr.getMnemonicString();
+		int instructionNumber;
+		switch (mnemonic) {
+		case "Call":
+			instr.setFlowOverride(FlowOverride.CALL);
+			instructionNumber = instr.getInt(NsisConstants.ARG1_OFFSET);
+			referenceManager.addMemoryReference(instr.getAddress(),
+					entriesBlock.getStart()
+							.add((instructionNumber - 1) * NsisConstants.INSTRUCTION_BYTE_LENGTH),
+					RefType.UNCONDITIONAL_CALL, SourceType.ANALYSIS, NsisConstants.ARG1_INDEX);
+			break;
+
+		case "Jmp":
+			instr.setFlowOverride(FlowOverride.BRANCH);
+			instructionNumber = instr.getInt(NsisConstants.ARG1_OFFSET);
+			referenceManager.addMemoryReference(instr.getAddress(),
+					entriesBlock.getStart()
+							.add((instructionNumber - 1) * NsisConstants.INSTRUCTION_BYTE_LENGTH),
+					RefType.UNCONDITIONAL_JUMP, SourceType.ANALYSIS, NsisConstants.ARG1_INDEX);
+			break;
+
+		case "Return":
+			instr.setFlowOverride(FlowOverride.CALL_RETURN);
```

Comment:
Same as Jmp here, add a call to `instr.setFallThrough(null);`

Reply:
good catch, I see the difference in the control flow now